docs/en/getting-started/get-started-docker.asciidoc

```diff
@@ -288,189 +293,121 @@ Settings (Windows).
 [discrete]
 ==== Prepare the environment
 
-Create the following Docker Compose and configuration files. These files are also
-available from the
+Create the following configuration files in a new, empty directory. These files
+are also available from the
 https://github.com/elastic/stack-docs/blob/master/docs/en/getting-started/docker/[elastic/stack-docs]
 repository on GitHub.
 
 --
 ifeval::["{release-state}"=="unreleased"]
 NOTE: Version {version} of {es} has not been released,
-so the sample compose and configuration files are not yet available for this version.
-See the {stack-gs-current}/get-started-docker.html[current version] for the latest sample files.
+so the sample Docker Compose and configuration files are not yet available for
+this version. See the {stack-gs-current}/get-started-docker.html[current version]
+for the latest sample files.
 endif::[]
+--
 
-* `instances.yml` identifies the instances you need to create certificates for.
-* `.env` sets environment variables to specify the {es} version and
-the location where the {es} certificates will be created.
-* `create-certs.yml` is a Docker Compose file that launches a container to generate the certificates
-for {es} and {kib}.
-* `elastic-docker-tls.yml` is a Docker Compose file that brings up a three-node {es} cluster
-and a {kib} instance with TLS enabled so you can see how things work.
-This all-in-one configuration is a handy way to bring up your first dev cluster before
-you build a distributed deployment with multiple hosts.
-
+--
 ifeval::["{release-state}"!="unreleased"]
-[discrete]
-===== `instances.yml`
-["source","yaml"]
-----
-include::docker/instances.yml[]
-----
 
 [discrete]
+[[docker-env-file]]
 ===== `.env`
+
+The `.env` file sets environment variables that are used when you run the
+`docker-compose.yml` configuration file. Ensure that you specify a strong
+password for the `elastic` and `kibana_system` users with the
+`ELASTIC_PASSWORD` and `KIBANA_PASSWORD` variables. These variable are
+referenced by the `docker-compose.yml` file.
+
 ["source","txt",subs="attributes"]
 ----
 include::docker/.env[]
 ----
 
 [discrete]
-===== `create-certs.yml`
-["source","txt"]
-----
-include::docker/create-certs.yml[]
-----
+[[docker-compose-file]]
+===== `docker-compose.yml`
 
-[discrete]
-===== `elastic-docker-tls.yml`
-["source","txt"]
+This `docker-compose.yml` file creates a three-node secure {es} cluster with authentication and network encryption enabled, and a {kib} instance securely connected to it.
+
+.Exposing ports on Linux
+****
+This configuration exposes port `9200` on all network interfaces. Given how
+Docker manipulates `iptables` on Linux, this means that your {es} cluster is
+publicly accessible, potentially ignoring any firewall settings. If you don't
+want to expose port `9200` and instead use a reverse proxy, replace `9200:9200`
+with `localhost:9200:9200` in the `docker-compose.yml` file. {es} will then only
```

Comment:
```suggestion
want to expose port `9200` to external hosts, set the ES_PORT value to something
like `127.0.0.1:9200` in the `.env` file. {es} will then only
```

Reply:
I'll make this change locally and include it in my next commit:

>If you don't want to expose port `9200` to external hosts, set the value for `ES_PORT` in 
the `.env` file to something like `127.0.0.1:9200`.